spec/views/curation_concerns/base/_relationships.html.erb_spec.rb

```diff
@@ -27,5 +51,74 @@
     it "labels the link using the presenter's #to_s method" do
       expect(page).not_to have_content 'foobar'
     end
+    it "shows the empty messages for parents" do
+      expect(page).not_to have_content "This Work is not currently in any collections."
+      expect(page).to have_content "This Work is not currently a part of any works."
+    end
+  end
+
+  context "when parents are present and no collections are present" do
+    let(:collection_presenters) { [generic_work] }
+    let(:page) { Capybara::Node::Simple.new(rendered) }
+    before do
+      allow(view).to receive(:contextual_path).and_return("/concern/generic_works/456")
+      allow(model_gw).to receive(:model_name).and_return("GenericWork")
```

Comment:
the `model_name` method should return an `ActiveModel::Name` object, not a string